x-pack/plugins/security_solution/public/management/components/search_exceptions/search_exceptions.tsx

```diff
@@ -0,0 +1,113 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { memo, useCallback, useState } from 'react';
+import { EuiFlexGroup, EuiFlexItem, EuiFieldSearch, EuiButton } from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+import { Item, PoliciesSelector } from '../policies_selector';
+import { ImmutableArray, PolicyData } from '../../../../common/endpoint/types';
+
+export interface SearchExceptionsProps {
+  defaultValue?: string;
+  placeholder: string;
+  hasPolicyFilter?: boolean;
+  policyList?: ImmutableArray<PolicyData>;
+  defaultExcludedPolicies?: string;
+  defaultIncludedPolicies?: string;
+  onSearch(query: string, includedPolicies?: string, excludedPolicies?: string): void;
+}
+
+export const SearchExceptions = memo<SearchExceptionsProps>(
```

Comment:
Cool. So this is now a "generic" search bar for anything we store in the List's Exceptions, right?

Reply:
Yeah. Well, this component works only with props, so we can reuse this easily in eventFilters and future different kind of exceptions (with or without policy selector as it's optional) and inside or outside the management tab.